src/aquarium-optimized/Aquarium.cpp

```diff
@@ -731,6 +740,20 @@ void Aquarium::printRecordFps()
     std::cout << std::endl << "End." << std::endl;
 }
 
+void Aquarium::printAvgFps()
+{
+    float avg  = 0.f;
+    bool valid = mFpsTimer.variance(&avg);
+    if (valid)
+    {
+        std::cout << "Avg FPS: " << ceil(avg) << std::endl;
+    }
+    else
+    {
+        std::cout << "Invalid value. The fps is unstable." << std::endl;
```

Comment:
If this is the final log, it's better to provide a friendly format for both of the user and the parser. If there's no valid value, simply print `Avg FPS: 0` so that the parser can use one single regular expression for both of valid and invalid result. Then you can explain something like "invalid value" to the user who checks the log.

Reply:
Done